data/remnant/remnant ships.txt

```diff
@@ -893,47 +893,32 @@ ship "Merganser"
 		licenses
 			Remnant
 		"cost" 6830000
-		"shields" 3000
-		"hull" 5000
+		"shields" 3500
+		"hull" 3000
 		"required crew" 1
 		"bunks" 3
 		"cargo space" 11
 		"fuel capacity" 600
-		"ramscoop" 1.75
-		"mass" 30
+		"ramscoop" 1.25
+		"mass" 90
 		"drag" 1.5
 		"heat dissipation" .9
 		"gaslining" 1
-		"outfit space" 219
-		"shield generation" 2.8
+		"outfit space" 180
```

Comment:
That's not a good response. No other Remnant ship remains crippled after ion storms. Now their latest and greatest ends up with worse performance than their centuries-old models?

Reply:
I do agree it shows we need to tweak something

Reply:
I've just tested this, and it _is_ true. All of those other ships recover very quickly from ion storms. The Starling, in particular, weathered 9 ion storms without being dropped below 25% energy. the 10th one finally dropped it to 0%, but it recovered to full energy in ~7 seconds (on 3x speed). 

The Gull recovered in 6.5 seconds. 

The Merganser in your configuration _never recovers_, because the mean time between ion storms is shorter than the recovery time. For the sake of testing, I immediately jumped out of EW just to see how long it does take. The answer is 61 seconds. Almost 10x the recovery time of the Gull and Starling. This is not appropriate behavior for a new Remnant ship.